### DISCUSSION TITLE
[Application] Filplus.info

### DISCUSSION THREAD

**Discussion by laurarenpanda** [filecoin-project/community#119] [Project Showcase]
## Project Description
[Filplus.info](https://filplus.info) is a website for recording DataCap allocation and verified deal transactions, developed by FileDrive Team.
 
### Homepage
The list of existing Filecoin Plus Notaries with relevant information.
![image](https://user-images.githubusercontent.com/74699939/112798446-7b6ef080-909f-11eb-9115-128fb17d0a73.png)

### Allocation Records
![image](https://user-images.githubusercontent.com/74699939/112800140-d4d81f00-90a1-11eb-9725-5e01ccbd5012.png)

### Verified Deal Transactions 
![image](https://user-images.githubusercontent.com/74699939/112800363-21bbf580-90a2-11eb-8618-aa96ca8767e4.png)

## Website
https://filplus.info

## Contact Us
Email: filedriveteam@outlook.com
Slack: https://filecoin.io/slack  
Twitter: https://twitter.com/FileDrive1
WeChat Official Account: [FileDrive](https://mp.weixin.qq.com/mp/profile_ext?action=home&__biz=MzkzMDE4ODg0MQ==&scene=124#wechat_redirect)

**Comment by jennijuju** [comment]
we would love to feature the latest updates of filplus.inofo in our showcase video. Is there any more updates you want to include?

**Reply by laurarenpanda** [reply]
Thnaks for asking. Yes, we'd love to.
For now, there’s no other update. We'll release next version maybe in next week.

**Comment by laurarenpanda** [comment]
![image](https://user-images.githubusercontent.com/74699939/119293386-e7776a80-bc84-11eb-901f-0e2dca67a1df.png)
![image](https://user-images.githubusercontent.com/74699939/119293693-8b611600-bc85-11eb-890d-6f011f5b7703.png)
![image](https://user-images.githubusercontent.com/74699939/119289507-4cc75d80-bc7d-11eb-9aab-c3ecce5761d2.png)

## Upgrades
For the week ending on Friday, May 21st
### filplus.info v2.1.0:
- Visual charts for data analysis
- Search functionality on Homepage
- Client pages with the following information: Notary name, Filecoin address, amout of allocated DataCap, and time of allocation
- Records of verified deal transactions: Deal ID links to the Fileoin explorer

**Comment by laurarenpanda** [comment]
### Notary Board on the homepage of filplus.info

![Notaries board 1](https://user-images.githubusercontent.com/74699939/128663580-55244545-e203-4374-babf-b762b612ba73.png)

![Notaries board 2](https://user-images.githubusercontent.com/74699939/128663606-f2ddb76b-ade5-4cbb-8c82-e548552a64fb.png)

## Upgrades
For the week ending on Friday, August 6th
- Information updating about new notaries from the second round of Notary elections

**Comment by laurarenpanda** [comment]
# Upgrades
For the week ending on Friday, October 1st
## [filplus.info](https://hackerlink.io/buidl/499?roundProj=499) v3.0.0 launched on Sep. 28th.
It is a very different version with a new UI design, statistics of three application methods of DataCap (including Notary allocation, large Datasets, and monthly automatic verification), and new features along with other bug fixes.

Significant new features including:
- Dashboard
  - More statistics and metrics about the Filecoin Plus program, including Notary allocation, large Datasets, and monthly automatic verification
  - Hyperlink jump to verify application page: *Issue: Client Datacap Allocation Request* and *Issue: Large Dataset Notary application* on GitHub, and DataCap faucet verify.glif.io
- Notaries
  - Information updating about new notaries from the second round of Notary elections
  - Search for a specific Notary by name/Filecoin address
  - Hyperlink jump to Notaries’ GitHub profile pages
  - Export Notary allocation record (display in the list, maximum 20)  as a CSV file
- Clients
  - Statistics of verified deals and DataCap distribution to cooperative miners by a client
  - Export DataCap receiving and usage record (display in the list, maximum 20) as a CSV file
- Large Datasets Application
  - The highest proportion of DataCap stored with one storage provider in each allocation
